tools/test.py

```diff
@@ -1452,15 +1487,15 @@ def wrap(processor):
 
   result = None
   def DoSkip(case):
-    return SKIP in case.outcomes or SLOW in case.outcomes
+    return SKIP in case.outcomes or SLOW in case.outcomes or (FLAKY in case.outcomes and options.flaky_tests == "skip")
```

Comment:
Styling: can we put the added condition on another line to avoid that long line?